CONTRIBUTING.md

```diff
@@ -36,6 +36,19 @@ Contributor Agreement (ECA) on file.
 For more information, please see the Eclipse Committer Handbook:
 https://www.eclipse.org/projects/handbook/#resources-commit
 
+## Project Contribution Guidelines
+
+ * Code should adhere to the style defined at https://github.com/eclipse-ee4j/ee4j.
+ * Contributions should be made via github pull requests at https://github.com/eclipse-ee4j/servlet-api/pulls 
+ * Authors of Pull Requests should seek review from committers outside of their own organisation and perspective.
+ * Committers and Contributors may review any Pull Requests or Commits, even if not requested or assigned.
+ * Pull Request by Committers that receive no review responses after a reasonable time, considering time zones and working days, can be interpreted as approved.
```

Comment:
How will this work in practice, as commiters can't merge their own commits?

Reply:
I assume that if this is adopted we want it common place, then we'd have to ask to have the requirement for a review to be removed from github.
Note that I think committers are able to merge locally and push to master or any branch, so they can always bypass the github check if they really want to.

Personally I don't mind having to seek review, so I'm happy not to have this.  But if others want it, I think the rest of the guidelines provide enough of a safety net that problems that arise could be handled.